### DISCUSSION TITLE
Landcape Lockscreen on AOSP

### DISCUSSION THREAD

**Discussion by nthebach** [phhusson/treble_experimentations#2335] [Q&A]
Hi my device is Huawei mediapad M5 pro 10.8 inches. When I installed AOSP 12.1 gsi on it the default orientation was portrait. When I set it to auto-rotate, the lockscreen stuck at portrait mode only. Also seems like android doesn't recognise it as a tablet so Youtube Music landscape mode was not available for my device as well.

**Comment by Hausemaster8281** [comment]
Feature request
+1
